config.yaml

```diff
@@ -27,7 +27,7 @@ params:
     disableThemeToggle: false
 
     ShowReadingTime: true
-    ShowShareButtons: true
```

Comment:
Removed because I find the button to share on 10 social media a bit too much, but we should include more prominently social media page somewhere (right now they are only in the home page introduction).

Reply:
Yeah I was looking at a test output and all the share buttons bugged me too.